corehq/apps/hqmedia/models.py

```diff
@@ -580,34 +584,35 @@ def _add_menu_media(item, **kwargs):
             }
             _add_menu_media(module, **media_kwargs)
 
-            for name, details, display in module.get_details():
-                if display and details.display == 'short' and details.lookup_enabled and details.lookup_image:
-                    media.append(ApplicationMediaReference(
-                        details.lookup_image,
-                        media_class=CommCareImage,
-                        **media_kwargs)
-                    )
-                # Icons in case-details
-                for column in details.get_columns():
-                    if column.format == 'enum-image':
-                        for map_item in column.enum:
-                            # iterate over icons of each lang
-                            icons = list(map_item.value.values())
-                            media.extend([ApplicationMediaReference(
-                                icon,
-                                media_class=CommCareImage,
-                                is_menu_media=True,
-                                **media_kwargs)
-                                for icon in icons
-                                if icon]
-                            )
-                # Print template
-                if display and details.display == 'long' and details.print_template:
-                    media.append(ApplicationMediaReference(
-                        details.print_template['path'],
-                        media_class=CommCareMultimedia,
-                        **media_kwargs)
-                    )
+            if module.module_type != 'report':
```

Comment:
is that a safe change to skip if module_type is 'report'? Looks out of scope of the aim of this PR.

Reply:
yea, it is a bit out of scope. The alternative was to fix a bunch of tests that need reports to exists. I checked the code though and report modules only ever use custom_xml:
* https://github.com/dimagi/commcare-hq/blob/5cd850522dd807edbe811d366223c31f9fc05f09/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py#L433
* https://github.com/dimagi/commcare-hq/blob/5cd850522dd807edbe811d366223c31f9fc05f09/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py#L122
* https://github.com/dimagi/commcare-hq/blob/5cd850522dd807edbe811d366223c31f9fc05f09/corehq/apps/app_manager/suite_xml/features/mobile_ucr.py#L280-L289

I'll take another look at the tests.

Reply:
reverted this commit and updated tests 87fb3defee6